hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpc/AsyncSocketBuilderTest.java

```diff
@@ -63,7 +63,8 @@ public void test_setReadHandler_whenAlreadyBuild() {
         builder.setReadHandler(new DevNullReadHandler());
         builder.build();
 
-        assertThrows(IllegalStateException.class, () -> builder.setReadHandler(new DevNullReadHandler()));
```

Comment:
What is the advantage here?

Reply:
Sonar says there should be one function call inside the lambda that can throw for better testing. Hence I extracted constructor.

Reply:
Ok.